### DISCUSSION TITLE
Lesson 9 modulo

### DISCUSSION THREAD

**Discussion by SCPassion** [Cyfrin/foundry-full-course-cu#722] [Q&A] [answered]
Dear guys,

I have a feeling that the comments in the video are wrong.
It should be:
```
//2 % 2 = 0; 3%2 = 1; 6%2 = 0; 7%2 = 1?
```
In video, the answer should be
```
// 2 %3 = 2; 2%6 = 2;;2 % 7 =2
```
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/141584825/a3511a90-8323-4dfb-8836-98b50b6150e3)

**Comment by theirrationalone** [comment, marked as answer]
Yep, You're absolutely right....

Recently someone  made a PR on that and now it's fixed into codebase, You can check by though!
